config/initializers/new_framework_defaults_7_0.rb

```diff
@@ -0,0 +1,143 @@
+# Be sure to restart your server when you modify this file.
+#
+# This file eases your Rails 7.0 framework defaults upgrade.
+#
+# Uncomment each configuration one by one to switch to the new default.
+# Once your application is ready to run with all new defaults, you can remove
+# this file and set the `config.load_defaults` to `7.0`.
```

Comment:
I set the defaults to 7.0, but keeping this file around for now just to see what the changes are.